README.md

```diff
@@ -170,28 +146,31 @@ If you want to edit any of the tests, you should edit [config.yml](.circleci/con
 Details about running tests locally are included below. This can be good to do if you
 change an input file in [_data](_data) and want to test it.
 
-#### 1. Test Map Entries
+#### 1. Test Jobs
 
-Other than previewing the site and ensuring that the coordinate shows up in the
-correct spot, you can run unit testing locally to confirm you have the minimum
-required data:
+Jobs are tested for correctness, meaning that all fields are entered, a date string
+is entered for the "expires" field, and the url is valid. You can run tests locally 
+like:
 
 ```bash
 $ cd tests
-$ python -m unittest test_mapdata
+$ python -m unittest test_jobs
 ```
 
-#### 2. Test Jobs
+#### 2. Test Map Entries
 
-Jobs are tested for correctness, meaning that all fields are entered, a date string
-is entered for the "expires" field, and the url is valid. You can run tests locally 
-like:
+**no longer in use**
```

Comment:
I'm not quite sure what the solution is. But looking at the README this seems confusing. I think it might be better to remove it entirely, or maybe put a date **Map test entries no longer in use as of April 2020**.

Reply:
okay I can remove entirely, I agree it could be confusing.

Reply:
All set.